### PR TITLE
#124: Adding allowed_lang configuration.

### PR DESCRIPTION
**Pull request by abpai94**
#124

### CODE REVIEW

File: conf/config.inc.php [hunk 1]
```diff
@@ -150,7 +150,8 @@
 $mail_priority = 3;
 
 # Language
-$lang ="en";
+$lang = "en";
+$allowed_lang = "";
```
Comment (coudot) [inline conf/config.inc.php:154]: Please initialize `$allowed_lang` as an array
```suggestion
$allowed_lang = array()
```

**Review by coudot — CHANGES_REQUESTED** [review]
Please review suggested changes and also update documentation (you can copy/paste from https://github.com/ltb-project/self-service-password/blob/master/docs/config_general.rst)

File: conf/config.inc.php (hunk 1 above)
Comment (abpai94) [reply conf/config.inc.php]: Fixed!

**Review by coudot — APPROVED** [review]

File: conf/config.inc.php [hunk 2]
```diff
@@ -150,7 +150,8 @@
 $mail_priority = 3;
 
 # Language
-$lang ="en";
+$lang = "en";
+$allowed_lang = array("");
```
Comment (coudot) [inline conf/config.inc.php:154]:
```suggestion
$allowed_lang = array();
```

**Comment by coudot** [comment]
Thanks!

**Pull request merged**